astroid/helpers.py

```diff
@@ -199,3 +199,44 @@ def class_instance_as_index(node):
     except exceptions.InferenceError:
         pass
     return None
+
+
+def object_len(node, context=None):
+    """Infer length of given node object
+
+    :param Union[nodes.ClassDef, nodes.Instance] node:
+    :param node: Node to infer length of
+
+    :raises AstroidTypeError: If an invalid node is given
```

Comment:
I can't quite parse this sentence, can you rephrase it?

Reply:
yeah